ansible_collections/arista/avd/roles/eos_designs/python_modules/inband_management/avdstructuredconfig.py

```diff
@@ -188,4 +146,64 @@ def route_maps(self) -> dict | None:
                     }
                 },
             }
-        }
+        ]
+
+    @cached_property
+    def _inband_management_parent_vlans(self) -> dict:
+        if not self.shared_utils.underlay_router:
+            return {}
+
+        svis = {}
+        subnets = []
+        peers = get(self._hostvars, f"avd_topology_peers..{self.shared_utils.hostname}", separator="..", default=[])
+        for peer in peers:
+            peer_facts = self.shared_utils.get_peer_facts(peer, required=True)
+            if (vlan := peer_facts.get("inband_mgmt_vlan")) is None:
+                continue
+
+            if (subnet := peer_facts.get("inband_mgmt_subnet")) in subnets:
+                continue
+
+            subnets.append(subnet)
+            svis[vlan] = subnet
+
+        return svis
+
+    def get_local_inband_mgmt_interface_cfg(self) -> dict:
```

Comment:
I know that because it is not a cached_property it is not picked up as a key for sturctured config but wondeirng if it would not help "the less knowledgeable reader" to have these as `_get_local_..`

Reply:
We talked about removing all the unneeded `_` from methods / vars here and there.
I can add `_` again if you prefer.

Reply:
ok - apologies 🤦 - let's not hold things back!